PartFourA.cpp

```diff
@@ -35,12 +34,6 @@ void writeEncodedFile(string arr[128]) {
 				}
 			}
 		}
-
-		for (counter; counter < 8; counter++) {

```

Comment:
yeah, these lines need to stay. This is what prints out the last character

Reply:
Oh, so strange. Okay, it is working now 'clear.' 

However, I noticed a problem. I was testing other data trying to figure out what was wrong and just tested that test clear file with 4A and 4B and it does not work properly.

Reply:
It's not allowing me to upload, but here is what is in test clear:

Reply:
Mary had a little lamb,
its fleece was white as snow.
And everywhere that Mary went
the lamb was sure to go.
He followed her to school one day
Which was against the rule;
It made the children laugh and play,
To see a lamb at school.
And so the teacher turned him out,
But still he lingered near;
And waited patiently about
Till Mary did appear
"What makes the lamb love Mary so?"
The eager chldren cry;
"Why, Mary loves the lamb, you know,"
The teacher did reply..
There was a	man. Who?

Reply:
It adds two 'oo's to the mix. Also, it does not recognize tab, but apparently neither does GitHub

Reply:
Mary had a little lamb,
its fleece was white as snow.
And everywhere that Mary went
the lamb was sure to go.
He followed her to school one day
Which was against the rule;
It made the children laugh and play,
To see a lamb at school.
And so the teacher turned him out,
But still he lingered near;
And waited patiently about
Till Mary did appear
"What makes the lamb love Mary so?"
The eager chldren cry;
"Why, Mary loves the lamb, you know,"
The teacher did reply..
There was aman. Who?oo

Reply:
Should we just use #include <bitset> ?

Reply:
@csreid: I'm looking at the bitset. Please let me know if you are getting these messages.

Reply:
What would `bitset` give us? I feel like that's a big change to make this late, but I haven't looked at its API

Reply:
Okay, do you know where the flaw is coming from?

Reply:
I don't see a flaw. When I run that test file through all the parts, I get the exact same data back,